ffi/src/log.rs

```diff
@@ -14,6 +14,11 @@ pub mod ffi {
     pub struct Log;
 
     impl Log {
+        /// # Panics
+        ///
+        /// FIXME: Panics if log directory creation fails.
+        ///
+        /// FIXME: Panics if tracing initialization fails.
```

Comment:
```suggestion
        /// Panics if log directory creation fails.
        ///
        /// Panics if tracing initialization fails.
        // FIXME: We should return an error instead, because panicking at the FFI boundary is unsafe.
```